app/src/androidTest/java/org/simple/clinic/TestClinicApp.kt

```diff
@@ -67,7 +67,7 @@ class TestClinicApp : ClinicApp() {
     // but we still have to provide a non-empty name for Room, otherwise it complains.
     return DaggerTestAppComponent.builder()
         .appModule(object : AppModule(this) {
-          override fun clock(): Clock = TestClock()
+          override fun clock(): Clock = TestUtcClock()
```

Comment:
Instrumented tests are not compiling anymore because the `DaggerTestComponent` hasn't been updated with the new signature of `AppModule`.

How did this get pushed though? Did you force push skipping the push hooks?

Reply:
Fixed